frontend/src/lib/api/private.ts

```diff
@@ -35,10 +39,11 @@ export async function listSelectieLijstKlasseChoices(
  * Retrieve zaaktypen from Open Zaak and return a value and a label per zaaktype. The label is the 'omschrijving' field
  * an the value is the URL. The response is cached for 15 minutes.
  */
-export async function listZaaktypeChoices() {
-  return cacheMemo("listZaaktypeChoices", async () => {
-    const response = await request("GET", "/_zaaktypen-choices/");
-    const promise: Promise<ZaaktypeChoice[]> = response.json();
-    return promise;
-  });
+export async function listZaaktypeChoices(
```

Comment:
Suggestion:

Use function arguments for `listReviewItems`.

```
listZaaktypeChoices(destructionListUuid, reviewPk)
```

Motivation:

I'd like to have a more standardizes interface for these API accessors. The main identifiers are typically a function argument and other parameters use a custom params type.

I think the function arguments are cleaner here.

P.S.:

Is there a reasons we identify a list by uuid and a review by pk?

Reply:
No, at some point we wanted to move to UUIDs, but we only go to migrating the destruction list and not all other resources :S